spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-lattice/src/main/java/org/springframework/cloud/dataflow/module/deployer/lattice/TaskModuleDeployer.java

```diff
@@ -76,7 +81,20 @@ public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
 		for (Map.Entry<String, String> entry : request.getDefinition().getParameters().entrySet()) {
 			environmentVariables.add(new EnvironmentVariable(entry.getKey(), entry.getValue()));
 		}
-
+		// Module launcher properties from Deployment properties
```

Comment:
All these duplicated handling may be moved to the SPI layer, to some helper method.
Maybe by turning `ModuleDeployer` into an abstract class + injecting the properties.